src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int index = new Random().nextInt(100);
+        Ball ball = new Ball(index, colorSupplier.getRandomColor());
+        return ball;
```

Comment:
Let's inline it and directly return the `new Ball` instead of creating a variable. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method).